tgstation.dme

```diff
@@ -6,180 +6,6 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
-#define FILE_DIR "config"
-#define FILE_DIR "config/title_music"
-#define FILE_DIR "config/title_music/sounds"
-#define FILE_DIR "config/title_screens"
```

Comment:
I'm not sure if there's a reason for this, but if there is there shouldn't be. It works on my machine.wav